### ISSUE TITLE
req: remember password and email (apple id)

### ISSUE THREAD

**Issue by gotexis** [altstoreio/AltStore#1167]
I don't understand why we need to input credentials every time we want to install AltStore, this is an open source project and it is safe to have this feature to remember credentails.

**Comment by microfx** [comment]
So is it so hard to do that???! Even IMPOSSIBLE? IQ 250 needed?

**Comment by microfx** [comment]
Hello?

**Comment by Ein-Tim** [comment]
@microfx First I'd suggest toning it down a bit, no feature was ever implemented because someone insulted the developer.

Second, there is basically no activity from anyone maintaining AltStore in this repo anymore, at least not since I started watching this repo ca. one year ago (or since AltStore Pal was published).

**Comment by microfx** [comment]
true... what do people use now then? It's just so annoying to sideload... I'm devastated

**Comment by Ein-Tim** [comment]
So there is https://github.com/SideStore & https://sideloadly.io/

Please note that I did not test any of them.
